src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -169,10 +170,10 @@ public static void WebSocket_Http_Duplex_BinaryStreamed()
         }
     }
 
-    [Fact]
+    [ConditionalFact(nameof(WebSockets_Supported), nameof(Root_Certificate_Installed))]
     [OuterLoop]
 #if !FEATURE_NETNATIVE
-    [ActiveIssue(420, PlatformID.AnyUnix)]
+    //[ActiveIssue(420, PlatformID.AnyUnix)]
```

Comment:
No, currently the WebSockets_Supported condition checks the platform and returns true for Windows, so it would run.  This is a perfect example where the websockets tests need to say "I need web socket support to run".  The alternative is listing the same one or more active issues on each web socket test which is less clear and duplicated information.

This gets to the heart of what [ConditionalFact] means.  To me it means "this test requires X to run" where X can be a feature, platform, machine configuration, environment variable, etc.  If there is one or more active issues preventing feature X from running, I think they should be listed in the implementation of the condition (e.g. WebSockets_Supported).  I can make that clearer in the code.